static/app/components/replays/walker/chevronDividedList.tsx

```diff
@@ -0,0 +1,59 @@
+import {ReactElement} from 'react';
+import styled from '@emotion/styled';
+
+import {IconChevron} from 'sentry/icons';
+import space from 'sentry/styles/space';
+
+type Props = {
+  items: ReactElement[];
+};
+
+function ChevronDividedList({items}: Props) {
+  return (
+    <List cols={items.length}>
+      {items.flatMap((item, i) => {
+        const li = <Item key={`${i}-item`}>{item}</Item>;
+
+        return i === 0
+          ? li
+          : [
+              <Item key={`${i}-chev`}>
+                <Chevron>
+                  <IconChevron color="gray300" size="xs" direction="right" />
+                </Chevron>
+              </Item>,
+              li,
+            ];
+      })}
+    </List>
+  );
+}
+
+const List = styled('ul')<{cols: number}>`
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: grid;
+  gap: ${space(1)};
+  grid-template-columns: ${p =>
+    `minmax(auto, max-content) repeat(${
+      (p.cols - 2) * 2 + 1
+    }, max-content) minmax(auto, max-content)`};
```

Comment:
This results in an extra template-column when there is only one segment to display. 

This means the `gap` will still take up space on the right, but the 2nd column itself with have a width of 0.